lib/ruby_lsp/document.rb

```diff
@@ -16,6 +16,7 @@ class LanguageId < T::Enum
     extend T::Generic
 
     ParseResultType = type_member
+    EmptyCache = T.let(Object.new.freeze, Object)
```

Comment:
Nit: `EmptyCache` feels like a class/module name. Maybe it could be `EMPTY_CACHE` instead?

Reply:
Ah, yeah, good point. It's more in the style of constants.